src/web/components/ApiKeyManagement/KeyHelper.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
+
+const yearInSeconds = 31556952;
```

Comment:
Doesn't really matter, but isn't the number of seconds in an average year 60*60*24*365.25=31557600? Or 31536000 ignoring leap days. maybe we could just use this so it's less magic: `const yearInSeconds = 60 * 60 * 24 * 365.25;`

Reply:
Agreed - writing out the math makes more sense for anyone looking at it later